pkg/visitors/extract.go

```diff
@@ -141,7 +141,7 @@ func (v *Extract) Visit(f uefi.Firmware) error {
 
 func init() {
 	var fileIndex uint64
-	RegisterCLI("extract", 1, func(args []string) (uefi.Visitor, error) {
+	RegisterCLI("extract", "extract the files to a directory", 1, func(args []string) (uefi.Visitor, error) {
```

Comment:
You seem to have inconsistent capitalization on the beginning of the help string. should we standardize on lower case for everything?

Reply:
yes